plugin-test/src/test/groovy/org/gradle/github/dependency/extractor/SampleProjectDependencyExtractorTest.groovy

```diff
@@ -56,6 +56,9 @@ class SampleProjectDependencyExtractorTest extends BaseExtractorTest {
         ]
     }
 
+    // Temporarily disable test that hangs on Gradle < 7.6
+    // TODO: Re-enable this test
+    @IgnoreIf({ System.getProperty("testGradleVersion") != null })
```

Comment:
Any idea what the problem is?

Reply:
Not yet. These versions were untested before, so this isn't a new problem. 
I did try older versions with a strategy using a `BuildService` and the test was still hanging.